boards/weact/stm32g030_core/Kconfig.defconfig

```diff
@@ -0,0 +1,22 @@
+# STM32G030 Core V1.0 board configuration
+
+# Copyright (c) 2025 Siratul Islam
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_WEACT_STM32G030_CORE
+
+config SPI_STM32_INTERRUPT
+	default y
+	depends on SPI
+
+# Kernel Options due to Low Memory (8k)
+config MAIN_STACK_SIZE
+	default 640 if SOC_STM32G030XX
+
+config IDLE_STACK_SIZE
+	default 200 if SOC_STM32G030XX
+
+config ISR_STACK_SIZE
+	default 512 if SOC_STM32G030XX
```

Comment:
Those `if SOC_STM32G030XX` should be removed as there is no doubt on this board.